modules/notice/main/index.scss

```diff
@@ -1,6 +1,7 @@
 .hx-notice {
   margin: 0.4em 0;
 
+  .hx-notice-head,
   .hx-notice-header {
```

Comment:
Might want to add a XXX [2.0.0] to remind us to remove the header version in 2.0.0